generative/maisi/scripts/augmentation.py

```diff
@@ -0,0 +1,555 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Hi @Can-Zhao , since there are many places that use index number from the label_dict. Can you add a TODO like below in the script to reference the label_dict when it gets uploaded, for example

```
"""
Functions to perform augmentation.
Reference: (TODO), a 132-class label dict that maps organ names and index. URL: TBD
"""

```

Reply:
Good idea! Added.